Apsitvarkom.Api/Controllers/PollutedLocationController.cs

```diff
@@ -0,0 +1,115 @@
+﻿using Apsitvarkom.Models;
+using Apsitvarkom.Models.DTO;
+using Microsoft.AspNetCore.Mvc;
+
+
+namespace Apsitvarkom.Api.Controllers  
+{
+    [ApiController]
+    [Route("/api/[controller]")]
+
+    public class PollutedLocationController : ControllerBase
+    {
+        /*temporary list PollutedLocations. Latter on we will have stored data in file or database. Then we will create
```

Comment:
I would suggest this to be deleted, as when our team is not big, we can track all our changes in our [project](https://github.com/orgs/vu-vibedosa/projects/1). If you were considering this comment to be useful for the lecturer, we can update him verbally.